5.process_CP_features/scripts/6.aggregation.py

```diff
@@ -58,12 +58,30 @@
             strata=["Metadata_Well", "Metadata_Time"],
             operation="median",
         )
+        aggregated_df = pd.merge(
+            aggregated_df,
+            fs_df[metadata_cols],
+            how="left",
+            on=["Metadata_Well", "Metadata_Time"],
+        )
```

Comment:
I do not know if you have done this in your notebook, but did you check the shape and the contents of your dataframe to see if it remained the same? if not, considered making a small function to check if nothing has changed after merging.